oak_functions/proto/server.proto

```diff
@@ -16,30 +16,30 @@
 
 syntax = "proto3";
 
-package oak.functions.server;
+package oak.session.stream.v1;
 
 option java_multiple_files = true;
-option java_package = "oak.functions.server";
+option java_package = "oak.session.stream.v1";
 
-message AttestedInvokeRequest {
+message StreamingRequest {
   bytes body = 1;
 }
 
-message AttestedInvokeResponse {
+message StreamingResponse {
   bytes body = 1;
 }
 
-service RemoteAttestation {
+service StreamingSession {
   // Creates a message stream for session key negotiation and encrypted payload exchange.
   //
   // The created message stream looks as follows:
-  // - Client->Server: `AttestedInvokeRequest` with a serialized `ClientHello` message.
-  // - Server->Client: `AttestedInvokeResponse` with a serialized `ServerIdentity` message.
-  // - Client->Server: `AttestedInvokeRequest` with a serialized `ClientIdentity` message.
-  // - Client->Server: `AttestedInvokeRequest` with a serialized `EncryptedData` message.
-  // - Server->Client: `AttestedInvokeResponse` with a serialized `EncryptedData` message.
+  // - Client->Server: `StreamingRequest` with a serialized `ClientHello` message.
+  // - Server->Client: `StreamingResponse` with a serialized `ServerIdentity` message.
+  // - Client->Server: `StreamingRequest` with a serialized `ClientIdentity` message.
+  // - Client->Server: `StreamingRequest` with a serialized `EncryptedData` message.
+  // - Server->Client: `StreamingResponse` with a serialized `EncryptedData` message.
```

Comment:
At some point this should move somewhere else; this proto only has the definition of the stream, not the structure of the packets inside the stream.